assets/js/atomic/blocks/product-elements/rating-counter/block.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "woocommerce/product-rating-counter",
+	"version": "1.0.0",
+	"title": "Product Rating Counter",
+	"description": "Display the review count of a product",
+	"attributes": {
+		"productId": {
+			"type": "number",
+			"default": 0
+		},
+		"isDescendentOfQueryLoop": {
+			"type": "boolean",
+			"default": false
+		},
+		"textAlign": {
+			"type": "string",
+			"default": ""
+		},
+		"isDescendentOfSingleProductBlock": {
+			"type": "boolean",
+			"default": false
+		},
+		"isDescendentOfSingleProductTemplate": {
+			"type": "boolean",
+			"default": false
+		}
+	},
+	"ancestor": [
+		"woocommerce/single-product",
+		"core/post-template",
+		"woocommerce/product-template"
```

Comment:
Considering it's registered using `registerBlockSingleProductTemplate` [here](https://github.com/woocommerce/woocommerce-blocks/pull/10114/files#diff-aaa6b369e78170b5ceff0586715f4e71bfac5598394864278efd045c4f243431R32), it cannot be inserted into Products nor Product Collection, so ancestor here can be removed, unless it's intended to be used in those blocks as well, then registering method has to be changed.

Reply:
> Link color is not applied in the frontend (size and alignment works fine).

I cannot reproduce that, color is applied for me 🤔 
<img width="668" alt="Screenshot 2023-07-11 at 09 59 18" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/c5370e8c-429c-4b3f-81e1-8ac9775bbed4">

> Product Rating Counter block cannot be inserted into Products and Product Collection (I'm not sure if that's expected or not). Worth to mention that Product Rating Stars CAN be inserted in both, so I assume Product Rating Counter should as well.

Let's see if I get it right this time 😅 I've pushed a change that allows inserting it in Products, Products Collection and also in the Single Product block and the Single Product template. I'll do the same on `Rating Stars`.